Yearn-agents/InstantWithdrawl/.env.example

```diff
@@ -0,0 +1 @@
+jsonRPC=
```

Comment:
Is this needed?

Reply:
yes. To show the name of the env variable